src/cryptocurrency.iced

```diff
@@ -18,4 +20,15 @@ exports.Cryptocurrency = class Cryptocurrency extends Base
   _v_customize_json : (ret) ->
     ret.body.cryptocurrency = @cryptocurrency
 
+  _v_check : ({json}, cb) ->
+    esc = make_esc cb, "SubkeyBase::_v_check"
+    await super { json }, esc defer()
```

Comment:
In this sort of situation, is there a risk that the parent constructor will start accepting some new keyword arg, and that we'll inadvertently strip it out here by explicitly listing the keywords we're passing down? Maybe a pattern like this would be better?

    _v_check : (args, cb) ->
        {json} = args
        await super args, esc defer()